data/sql/updates/pending_db_world/rev_1587839068992670400.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1587839068992670400');
+
+DELETE FROM `spell_script_names` WHERE `ScriptName`="spell_item_eggnog";
```

Comment:
For spell_script_names always delete using both id and name. I know in this case it won't matter for this particular issue, but several spell_id's can have the same ScriptName, that is why we always should delete using both.
```suggestion
DELETE FROM `spell_script_names` WHERE `spell_id`=21149 AND `ScriptName`="spell_item_eggnog";
```

Reply:
an example of this is 
```sql
SELECT * FROM spell_script_names WHERE scriptname = 'spell_gen_no_offhand_proc'
``` 
It has two spell_id's with the same scriptname

Reply:
Good observation!